Cargo.toml

```diff
@@ -1,23 +1,25 @@
 [package]
 name = "bcs"
 version = "0.1.4"
-authors = ["Diem <opensource@diem.com>"]
 description = "Binary Canonical Serialization (BCS)"
-repository = "https://github.com/diem/bcs"
-homepage = "https://diem.com"
+repository = "https://github.com/zefchain/bcs"
```

Comment:
So we've forked from diem repository but setting it to zefchain - maybe we should form directly from zefchain if we are using their version?